src/Security/test/AuthSamples.FunctionalTests/AuthSamples.FunctionalTests.csproj

```diff
@@ -4,6 +4,7 @@
     <TargetFramework>netcoreapp3.0</TargetFramework>
     <IsSampleProject>false</IsSampleProject>
     <PlatformTarget>AnyCPU</PlatformTarget>
+    <RunOnOSX1014>true</RunOnOSX1014>
```

Comment:
This is just for testing purposes. Need to actually identify a list of tests that need to be run on OSX 10.14

Reply:
I say we identify the list of tests after we unblock helix.

Reply:
Yup, and the results before I accidentally closed the PR illustrates that this mechanism of enabling test runs on OSX.10.14  works: https://mc.dot.net/#/user/aspnetcore/pr~2Faspnet~2Faspnetcore/ci/20190604.32.

Reply:
Okay actually there must be at least one test that runs on each queue. I'm going to just turn this test on as a placeholder until we determine which the set of tests that actually need to be tested on this platform.

Reply:
Doesn't that defeat the purpose of this change? This will still cause Helix to hang for 4 hours.

Reply:
The point of this change is to run a smaller set of tests so we don't overwhelm the queue. But it doesn't mean we want to completely remove coverage.